debian/rules

```diff
@@ -40,6 +40,14 @@ configure-stamp:
 	./configure --prefix=/usr --localstatedir=/var \
 	  --sysconfdir=/etc $(extraopts) $(confflags) \
 	  $(CEPH_EXTRA_CONFIGURE_ARGS)
+
+	# Wheezy and Precise do not support tracepoints
```

Comment:
All other "modern" Debian-based OSes have a functional LTTng-UST. Since only hammer needs to build on these older distros, this fix only affects the deb building process for those two releases(since autoconf detects that LTTng is broken).